### ISSUE TITLE
Repeated ‘\frac macro substitution failed’ warning—any risk to reward correctness?

### ISSUE THREAD

**Issue by Wizard759** [Shenzhi-Wang/Beyond-the-80-20-Rule-RLVR#24]
Hi @author,
I’m seeing the following warning every time I run the code:
```
(TaskRunner pid=150767) WARNING:2025-12-23 20:21:52,211:WARNING: Error in configuration: macro '\frac' failed its substitution!
(TaskRunner pid=150767) WARNING:2025-12-23 20:21:53,799:WARNING: Error in configuration: macro '\frac' failed its substitution!
(TaskRunner pid=150767) WARNING:2025-12-23 20:21:54,136:WARNING: Error in configuration: macro '\frac' failed its substitution!
(TaskRunner pid=150767) WARNING:2025-12-23 20:21:55,437:WARNING: Error in configuration: macro '\frac' failed its substitution!
(TaskRunner pid=150767) WARNING:2025-12-23 20:21:55,437:WARNING: Error in configuration: macro '\frac' failed its substitution!
(TaskRunner pid=150767) WARNING:2025-12-23 20:21:57,048:WARNING: Error in configuration: macro '\frac' failed its substitution!
(TaskRunner pid=150767) WARNING:2025-12-23 20:21:57,796:WARNING: Error in configuration: macro '\frac' failed its substitution!
(TaskRunner pid=150767) WARNING:2025-12-23 20:21:57,845:WARNING: Error in configuration: macro '\frac' failed its substitution!
```
Have you encountered this before?
Could this warning silently affect the correctness of the rewards or any other critical outputs?
Any guidance on how to suppress or fix it would be greatly appreciated!
Thanks in advance for your time.

**Comment by caiyuchen-ustc** [comment]
+1

**Comment by 12cyt261** [comment]
Is it solved?
